src/main/java/com/code_intelligence/jazzer/driver/Opt.java

```diff
@@ -144,27 +144,27 @@ public final class Opt {
     OptParser.failOnUnknownArgument();
 
     if (help) {
-      err.println(OptParser.getHelpText());
+      Log.println(OptParser.getHelpText());
       exit(0);
     }
     if (version) {
-      err.printf("Jazzer v%s", JAZZER_VERSION);
+      Log.error("Jazzer v" + JAZZER_VERSION);
```

Comment:
Should be `Log.println` as it's no error.

Reply:
Fixed.